### ISSUE TITLE
Better way to append a DOMNode

### ISSUE THREAD

**Issue by tolgaulas** [servo-php/fluidxml#5]
Is there a way (better?) then the below?

$xml    = new SimpleXMLElement($inXML);
$domxml = dom_import_simplexml($xml);
$stripedXML =  $domxml->ownerDocument->saveXML($domxml->ownerDocument->documentElement);
$outXML = new FluidXml();
$outXML->add('firstlevel',true)->appendXml($inXML);

**Comment by daniele-orlando** [comment]
Good point.
In my workflow it's almost an edge case, but I understand it can be a reasonable situation.
Given your suggestion I will commit an `->appendXml()` patch to recognize SimpleXMLElement e domdocument/domnode.

**Comment by daniele-orlando** [comment]
@tolgaulas c21d80a87a185675816a4cb5a3195dee54f468fa improves `appendXml()` in a way that it recognize `DOMNode`/`DOMNodeList`/`SimpleXMLElement`/`FluidXML` instances.

In the last release (1.9) `appendXml()` has been merged with `appendChild()`/`add()`.
Your code become:
```
$xml = new SimpleXMLElement($inXML);
$outXML = new FluidXml();
$outXML->add('firstlevel',true)->add($xml);
```

**Comment by daniele-orlando** [comment]
Closed by 39053c141f6e44373a33c4ac7967554b79ad5396.
